### ISSUE TITLE
gpg: keyserver receive failed: Server indicated a failure

### ISSUE THREAD

**Issue by giri-uno** [mongo-express/mongo-express#842]
Hi all,

I pulled the latest master branch and try to build the docker image. But While building I am getting the following error 

![image](https://user-images.githubusercontent.com/101640839/164603473-01302baa-30d4-4db7-aa98-f66644d9f6ab.png)
Kindly let me know what exactly we need to change in docker file to make the docker working as expected

**Comment by kavaldeep** [comment]
did you solve by any chance ?

**Comment by giri-uno** [comment]
Hi kavaldeep,

No

**Comment by BlackthornYugen** [comment]
@giri-uno you might find the dockerfile in https://github.com/mongo-express/mongo-express-docker useful. If you change to alpine 'tiny' can be installed with `RUN apk add --no-cache tini` -- on debian it is currently installed by adding a gpg key from a keyserver at ha.pool.sks-keyservers.net. 

```
gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 6380DC428747F6C393FEACA59A84159D7001A4E5 \
gpg --batch --verify /usr/local/bin/tini.asc /usr/local/bin/tini
```

It looks like [sks-keyservers is depricated](https://unix.stackexchange.com/questions/656205/sks-keyservers-gone-what-to-use-instead)

**Comment by BlackthornYugen** [comment]
Oh! It looks like this key is available from `keyserver.ubuntu.com`. 

```
$ time gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 6380DC428747F6C393FEACA59A84159D7001A4E5
gpg: keyserver receive failed: Server indicated a failure
gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys   0.01s user 0.01s system 1% cpu 1.423 total

$ time gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 6380DC428747F6C393FEACA59A84159D7001A4E5
gpg: keyserver receive failed: Server indicated a failure
gpg --keyserver ha.pool.sks-keyservers.net --recv-keys   0.01s user 0.01s system 11% cpu 0.167 total

$ time gpg --batch --keyserver keyserver.ubuntu.com --recv-keys 6380DC428747F6C393FEACA59A84159D7001A4E5
gpg: key 9A84159D7001A4E5: "Thomas Orozco <thomas@orozco.fr>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1
gpg --batch --keyserver keyserver.ubuntu.com --recv-keys   0.01s user 0.01s system 5% cpu 0.434 total
```

**Comment by BlackthornYugen** [comment]
Looks like #804 fixes this

**Comment by BlackthornYugen** [comment]
@giri-uno #804 has now been merged. Can you confirm if this is solved for you?

**Comment by rtritto** [comment]
Will be fixed with #583

FYI @BlackthornYugen
